src/Tizen.NUI/src/internal/Common/ReusablePool.cs

```diff
@@ -0,0 +1,163 @@
+/*
+ * Copyright(c) 2025 Samsung Electronics Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+using System;
+using System.Collections.Generic;
+
+namespace Tizen.NUI
+{
+    internal static class ReusablePool<T> where T : Disposable, new()
+    {
+        private const int MaxPoolSize = 3;
+        private static readonly Stack<T> pool = new ();
```

Comment:
```suggestion
        private static readonly ConcurrentStack<T> pool = new ();
```

Minor. Can't we use `ConcurrentStack` instead of `Stack`? Since `Tizen.NUI.Disposable` classes are thread safe.

Reply:
Great idea! I also thought about it but
(1) since this reusable pool should be used in event thread, (It is used when manipulating View's property which should be done in event thread)
(2) and the `ConcurrentStack` is less-than-ideal performance compared with simple stack
So I concluded not to use it for now. Instead I'll add descriptions to notify this it not thread-safe!